### ISSUE TITLE
ssh: must specify HostKeyCallback

### ISSUE THREAD

**Issue by hectorqin** [rapidloop/rtop#35]
I got this problem when i run this code on my macOS 10.12.3,  but i have fixed it by editing  the  file of  sshhelper.go

about line 214:
``` go
config := &ssh.ClientConfig{
	User: user,
	Auth: auths,
	HostKeyCallback: func(hostname string, remote net.Addr, key ssh.PublicKey) error {
            return nil
        },
}
```

**Comment by kron4eg** [comment]
Yeah, this caused by recent changes in x/crypto/ssh

**Comment by rotelok** [comment]
the fix works for me, was getting the exact same error

**Comment by mdevan** [comment]
Fixed in c49441b.
